hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1179,15 +1229,22 @@ public void start() throws IOException {
       // Allow OM to start as Http Server failure is not fatal.
       LOG.error("OM HttpServer failed to start.", ex);
     }
-    omRpcServer.start();
-    isOmRpcServerRunning = true;
 
+    if (!prepareForUpgrade) {
+      omRpcServer.start();
+      isOmRpcServerRunning = true;
+    }
```

Comment:
During prepareForUpgrade, the RPC server is not stared. So we should also have the corresponding command to trigger to restart RPC server.

Reply:
As we discussed this with @avijayanhwx during internal design discussions, after OM is started in prepareForUpgrade mode, it will tear down, when the last transaction is applied from the raft log, and a snapshot is taken in raft, so with that the OM reached a state when all transactions are applied and none needs to be applied after the next startup.

This is to ensure that all the transactions are applied with the code that was there when the transactions arrived in, so with that we can ensure consistency of the state of different OM instances.

After this is finished, and OM tear down from prepareForUpgrade, one will need a normal startup of OM to bring it up again, and at that time the RPC server will start properly.

Reply:
Okay, enable RPC server via next startup makes sense to me.